app/views/admin/conferences/edit.html.haml

```diff
@@ -22,11 +22,11 @@
         = f.input :timezone, as: :time_zone, hint: 'The conference time zone'
         = f.input :start_date, as: :string, input_html: { id: 'conference-start-datepicker', readonly: 'readonly' }
         = f.input :end_date, as: :string, input_html: { id: 'conference-end-datepicker', readonly: 'readonly' }
-        = f.input :start_hour, input_html: {size: 2, type: 'number', min: 0, max: 23}
-        = f.input :end_hour, input_html: {size: 2, type: 'number', min: 1, max: 24}
+        = f.input :start_hour, input_html: { size: 2, type: 'number', min: 0, max: 23}, hint: 'Changing the conference hours will unschedule your scheduled events.'
+        = f.input :end_hour, input_html: { size: 2, type: 'number', min: 1, max: 24}, hint: 'Changing the conference hours will unschedule your scheduled events.'
       = f.inputs name: 'Registrations' do
         = f.input :registration_limit, as: :number, in: 0..9999, hint: 'Limit the number of registrations to the conference (0 no limit). Please note that the registration limit doesn\'t apply to speakers of confirmed events (they will still be able to register even if it has been reached). You currently have ' + pluralize(@conference.registrations.count, 'registration')
       = f.inputs name: 'Booths' do
         = f.input :booth_limit, as: :number, in: 0..9999,
         hint: 'Booth limit is the maximum number of booths that you can accept for this conference. By setting this number (0 no limit) you can be sure that you are not going to accept more booths than the conference can accommodate. You currently have ' + pluralize(@conference.booths.accepted.count, 'accepted booth') +'.'
-      = f.action :submit, as: :button, button_html: {class: 'btn btn-primary'}
+      = f.action :submit, as: :button, button_html: {class: 'btn btn-primary', data: { confirm: 'Are you sure you want to proceed?' } }
```

Comment:
Let's include on that message the useful information as well. Eg.

> 20 event are going to be unscheduled. Are you sure you want to proceed?

Reply:
Yes, it will be better 😄